post-deploy-tests/fixtures/write-activity-log.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Records": [
+    {
+      "messageId": "19dd0b57-b21e-4ac1-bd88-01bbb068cb78",
+      "receiptHandle": "MessageReceiptHandle",
+      "body": "{\"event_id\":\"event-id-1\",\"event_type\":\"AUTH_AUTH_CODE_ISSUED\",\"session_id\":\"session-id\",\"user_id\":\"post-deploy-tests\",\"timestamp\":1720465374,\"client_id\":\"client-id\",\"reported_suspicious\":false}",
+      "attributes": {
+        "ApproximateReceiveCount": "1",
+        "SentTimestamp": "1523232000000",
+        "SenderId": "123456789012",
+        "ApproximateFirstReceiveTimestamp": "1523232000001"
+      },
+      "messageAttributes": {},
+      "md5OfBody": "e359050680237e70f4b71093185eca98",
+      "eventSource": "aws:sqs",
+      "eventSourceARN": "arn:aws:sqs:us-east-1:123456789012:MyQueue",
+      "awsRegion": "us-east-1"
```

Comment:
Would it matter if we use this instead of eu-west-2?

Reply:
It doesn't seem to matter - I think this is indicating that the event originated in `us-east-1` but the lambda is still invoked in `eu-west-2` where it's deployed.

Reply:
Let's try and keep eu-west-2 everywhere.  It doesn't matter, but it's just easier cognitively to know the vast majority is eu-west-2.